src/main/java/spoon/support/Internal.java

```diff
@@ -13,10 +13,10 @@
 import java.lang.annotation.Target;
 
 /**
- * Tells that a class or method is not in the public API (even if it has Java visibility "public")
+ * Tells that an element is not in the public API (even if it has Java visibility "public")
  * Required because package-visibility is too coarse-grained.
  */
 @Retention(RetentionPolicy.RUNTIME)
-@Target({ ElementType.METHOD, ElementType.TYPE })
+@Target({ElementType.CONSTRUCTOR, ElementType.METHOD, ElementType.TYPE })
```

Comment:
This was needed as I wanted to mark the new constructor of the ZipFile internal.